src/metabase/models/pulse.clj

```diff
@@ -192,7 +193,7 @@
   "Update an existing `Pulse`, including all associated data such as: `PulseCards`, `PulseChannels`, and `PulseChannelRecipients`.
 
    Returns the updated `Pulse` or throws an Exception."
-  [{:keys [id name cards channels] :as pulse}]
+  [{:keys [id name cards channels skip_if_empty?] :as pulse}]
```

Comment:
@camsaul I changed that back because it's inconsistent with other fields of the same map; see https://github.com/metabase/metabase/blob/fa6f052f726e36c851029a6a9c32791057d36be7/test/metabase/models/pulse_test.clj#L184